book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] I think this method for adding rentals should take in the `person` and the `date` and the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.` like this :point_down:
  
  ```ruby
  # *A method for adding rentals should be present.
  def add_rental(person, date)
      # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
      Rental.new(date, self, person)
    end
  ```